### DISCUSSION TITLE
Change cursor position on click and selection?

### DISCUSSION THREAD

**Discussion by persiklover** [vercel/hyper#6566] [Q&A]
Does hyper have an option to put a cursor under the mouse position after click/select?
If not, can we have it in the future?

Here's what I'm talking about:

https://user-images.githubusercontent.com/38048337/170863867-2dd8950e-0b01-442f-8e0f-4e3358ddfb03.mp4

**Comment by shinmai** [comment]
As far as I can tell, Hyper does not have mouse events support on Windows. Based on some mouse related Issues tickets, it apparently does work on Mac. The oldest issue about missing mouse event support on Windows dates back to 2017, so even though they're not marked wontfix, I'm assuming this isn't something that's being worked on or will be anytime soon.

This didn't used to be an issue for me, but with some changes in the work I need to do lately, I may have to start looking for a different terminal emulator, honestly.
